code/modules/mob/mob.dm

```diff
@@ -1062,11 +1062,14 @@
 	return 1
 
 // Let simple mobs press buttons and levers but nothing more complex.
-/mob/proc/has_dexterity(var/dex_level)
-	. = dex_level <= DEXTERITY_SIMPLE_MACHINES
-
-/mob/proc/check_dexterity(var/dex_level, var/silent)
-	. = has_dexterity(dex_level)
+/mob/proc/get_dexterity(var/silent = FALSE)
+	var/decl/species/my_species = get_species()
+	if(my_species)
+		return my_species.get_manual_dexterity()
+	return DEXTERITY_BASE
+
+/mob/proc/check_dexterity(var/dex_level = DEXTERITY_FULL, var/silent = FALSE)
+	. = !!(get_dexterity(silent) & dex_level)
```

Comment:
Are we sure this doesn't check if it has *any* of the bitflags rather than all? 10 & 11 -> 10, !!10 -> TRUE. Should probably be `(get_dexterity(silent) & dex_level) == dex_level`?

Reply:
The default check appears to be `DEXTERITY_FULL`, is that supposed to be "has any dexterity" rather than "has every dexterity"?